src/hooks/useAccommodationsAndOffers/useAccommodationsAndOffersHelpers.ts

```diff
@@ -25,28 +26,37 @@ export const useAccommodationsAndOffersHelpers = () => {
     });
   };
 
-  // normalize offers hook
   const normalizeOffers = useCallback(
-    (offers: Record<string, Offer>): OfferRecord[] => {
+    (
+      offers: Record<string, Offer> | undefined,
+      accommodations: Record<string, WinAccommodation> | undefined
+    ): OfferRecord[] => {
       if (!offers) return [];
 
-      const normalizedData = Object.entries(offers).map<OfferRecord>(([key, value]) => ({
+      const offersArray = Object.entries(offers).map<OfferRecord>(([key, value]) => ({
         id: key,
         ...value
       }));
 
-      let normalizedOffers: OfferRecord[];
+      let offersWithPreferredCurrency: OfferRecord[];
 
       if (preferredCurrencyCode) {
-        normalizedOffers = getOffersWithPreferredCurrency(
-          normalizedData,
+        offersWithPreferredCurrency = getOffersWithPreferredCurrency(
```

Comment:
The offers may not always include the preferred currency and as such, naming this way could be misleading